pkgs/development/python-modules/loguru/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, isPy2, pytest, colorama }:
+
+buildPythonPackage rec {
+  pname = "loguru";
+  version = "0.3.0";
+  disabled = isPy2;
```

```suggestion
  
  disabled = isPy27;
```

fix eval, whitespace